themes/Coinsence/views/space/space/_layout.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use humhub\modules\space\widgets\Header;
+use humhub\modules\space\widgets\Menu;
+use humhub\modules\space\widgets\SpaceContent;
+use humhub\widgets\FooterMenu;
+
+/**
+ * @var \humhub\modules\ui\view\components\View $this
+ * @var \humhub\modules\space\models\Space $space
+ * @var string $content
+ */
+
+/** @var \humhub\modules\content\components\ContentContainerController $context */
+$context = $this->context;
+$space = $context->contentContainer;
```

Comment:
add var annotations here too

Reply:
this is their default template, also there are some var annotation above